static/js/upload_widget.js

```diff
@@ -152,7 +152,7 @@ var upload_widget = (function () {
                     input_error.show();
                     clear();
                 } else {
-                    accept(file);
+                    accept();
```

Comment:
You mean the one on [line 33](https://github.com/xiemaisi/zulip/blob/85c3f59292d8ef575b3627ad7aeaa9f36c144dc2/static/js/upload_widget.js#L33)? That one isn't in scope here. This call refers to the function defined on [line 118](https://github.com/xiemaisi/zulip/blob/85c3f59292d8ef575b3627ad7aeaa9f36c144dc2/static/js/upload_widget.js#L118), which does not take any arguments.